<launch>
    <param name="use_sim_time" value="True"/>

    <!-- <node pkg="team2_package" type="carla_route_saver.py" name="carla_route_saver" output="screen"/> -->

    <!--<node pkg="team2_package" type="controller" name="controller" output="screen"/>-->

    <!--<node pkg="team2_package" type="gps_to_xyz" name="gps_to_xyz" output="screen"/>-->
    <node pkg="team2_package" type="global_planner" name="global_planner" output="screen"/>

    <!-- <include file="/home/ailab/Desktop/CARLA_Leaderboard_2.0/CARLA_Leaderboard_20/team_code/catkin_ws/src/team2_package/launch/filter_ver_2.launch"/> -->

    <!--<node pkg="team2_package" type="keyboard_control" name="keyboard_control" output="screen"/>-->

    <!--<node name="rviz" pkg="rviz" type="rviz" args="-d /home/ailab/Desktop/CARLA_Leaderboard_2.0/CARLA_Leaderboard_20/team_code/catkin_ws/src/team2_package/src/perception/test.rviz"/>-->

    <!--<node pkg="team2_package" type="localizer" name="localizer" output="screen"/>-->
</launch>
